lighthouse-core/test/gather/gatherers/start-url-test.js

```diff
@@ -154,20 +154,7 @@ describe('Start-url gatherer', () => {
 
     return startUrlGatherer.afterPass(options, tracingData)
       .then(artifact => {
-        assert.equal(artifact.explanation, 'ERROR: start_url string empty');
-      });
-  });
-
-  it('returns an error when origin is not the same', () => {
```

Comment:
not an error, a warning

Reply:
is it easy to convert to a test for the warning?

Reply:
> is it easy to convert to a test for the warning?

the gatherer was set up to either fail and return an explanation or succeed and have no explanation, so making it not fail in that case means no more warning from the manifest parser. But both `webapp-install-banner` and `offline-start-url` both check the warning straight from the `Manifest` artifact (and both have tests)